README.md

```diff
@@ -83,13 +83,13 @@ Participants will receive zoom invitations via email.
 
 | **Theme** | **Duration** | **UTC Time** |
 |-----------|--------------|--------------|
-| Second issue - break-out group discussion | 15 min | 20:00 - 20:15 |
-| Second issue - group reports | 10 min | 20:15 - 20:25 |
-| Second issue - local development | 25 min | 20:25 - 20:50 |
+| [Writing unit tests](activities/writing-unit-tests.md) | 20 min | 20:00 - 20:20 |
+| Second issue - break-out group discussion | 15 min | 20:20 - 20:35 |
+| Second issue - group reports | 15 min | 20:35 - 20:50 |
 | Break | 10 min | 20:50 - 21:00 |
-| Second issue - submitting through issues/pull requests | 50 min |21:00 - 21:50 |
+| Second issue - local development to work through issues/pull requests | 50 min | 21:00 - 21:50 |
 | Break | 10 min | 21:50 - 22:00 |
-| Second issue - incorporating suggestions | 50 min | 22:00 - 22:50 |
+| Second issue - incorporating suggestions from reviewers | 50 min | 22:00 - 22:50 |
 | Break | 10 min | 22:50 - 23:00 |
 | Helping maintain software | 45 min | 23:00 - 23:45 |
```

Comment:
The helping maintain software section might be a bit long. My original plan was to lump testing into this section, so I think we should shorten it now.
```suggestion
| Second issue -  continue working on contributions | 30 min | 23:00 - 23:30 |
| Maintaining software | 10 min | 23:30 - 23:23:40 |
| Resources for further learning | 5 min | 23:40 - 23:45 |
```